MinnPost/object-sync-for-salesforce
# 140/127 fieldmap data structure

## What does this PR do?

This fixes #140 and #127. Thus it changes the field type of `fields` to `longtext` instead of `text`, and it adds a `version` column. To prevent database errors, it only tries to load (or save) the version column for installations running at least version 1.2.5.

## How do I test this PR?

The most thorough way is like this:

1. Deactivate the plugin (don't let it delete the data though).
2. Reactivate the plugin.
3. Do any kind of edit on a single fieldmap and make sure the database now has a value for version.
4. Add a ton of fields to the fieldmap.
